components/store.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { createContext, useContext, useReducer } from 'react';
```

Comment:
Denne filen blir ikke brukt, men jeg hadde ikke hjerte til å slette den etter at vi knotet så mye med typene. Kanskje jeg får bruk for store snart?

Reply:
Den største glede i livet er å slette kode man en gang skrev. Neste gang vet du ca hvordan du skal skrive den, sant?

Reply:
Jeg er egentlig enig. Jeg sletter den!